client/ayon_core/tools/sceneinventory/view.py

```diff
@@ -735,42 +749,58 @@ def _show_version_dialog(self, item_ids, active_repre_id):
         container_items_by_id = self._controller.get_container_items_by_id(
             item_ids
         )
-        repre_ids = {
-            container_item.representation_id
-            for container_item in container_items_by_id.values()
-        }
-        repre_info_by_id = self._controller.get_representation_info_items(
-            repre_ids
-        )
+        repre_ids_by_project = collections.defaultdict(set)
+        for container_item in container_items_by_id.values():
+            repre_id = container_item.representation_id
+            project_name = container_item.project_name
+            repre_ids_by_project[project_name].add(repre_id)
+
+        versions = set()
```

Comment:
Anything using this `set` down the line wouldn't know which version id the project belongs to. This should be per project because each project may have overlapping version ids. Also, please name this variable better. What is this storing? I assume version ids? If so, name this `version_ids` - if not then I'm not sure what this will be holding.